src/org/jitsi/impl/neomedia/RawPacket.java

```diff
@@ -122,6 +122,21 @@ public static boolean isPacketMarked(byte[] buffer, int offset, int length)
     }
 
     /**
+     * Perform checks on the packet represented by this instance and
+     * return <tt>true</tt> if it is found to be invalid. A return value of
+     * <tt>false</tt> does not necessarily mean that the packet is valid.
+     *
+     * @return <tt>true</tt> if the RTP/RTCP packet represented by this
+     * instance is found to be invalid, <tt>false</tt> otherwise.
+     */
+    public static boolean isInvalid(byte[] buffer, int offset, int length)
+    {
+        return (buffer == null)
+            || (buffer.length < offset + length)
+            || (length < FIXED_HEADER_SIZE);
```

Comment:
I don't think we should use this method for RTCP packets (I'll make sure I'm not using this anywhere) + maybe we should rethink the structure of this class..